lib/application.js

```diff
@@ -211,7 +210,7 @@ app.use = function use(fn) {
     }
   }
 
-  var fns = flatten(slice.call(arguments, offset));
+  var fns = slice.call(arguments, offset).flat()
```

Comment:
You need `flat(Infinity)` for the same behavior.

Reply:
Keep in mind that changing this is a breaking change, so you'd want to be making the change in Express.js v5 and not to the current v4 release.

Reply:
Indeed, based on the test results, these changes are intended for a future version of Express.

Sorry for wanting to remove your dependency! 🤣